src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,45 @@
 package core.basesyntax.impl;
 
+import core.basesyntax.Pair;
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private final Pair<K, V>[] pair;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

it will be initialized by 0 by default